yt_dlp/extractor/facebook.py

```diff
@@ -493,14 +509,14 @@ def process_formats(info):
 
         def extract_relay_data(_filter):
             return self._parse_json(self._search_regex(
-                r'handleWithCustomApplyEach\([^,]+,\s*({.*?%s.*?})\);' % _filter,
+                r'data-sjs>({.*?%s.*?})</script>' % _filter,
```

Comment:
same

Reply:
This one is more tricky. There is no obvious negative pattern that could optimise it since `_filter` is unknown too. So potential alternative would be to grab all jsons with `r'data-sjs>([^<]+)</script>'` regex, then find the first one matching `_filter`. Again, not sure if this worth considering for such small files.